platform/linux-generic/include/odp_buffer_internal.h

```diff
@@ -104,17 +104,10 @@ struct odp_buffer_hdr_t {
 	/* User area pointer */
 	void    *uarea_addr;
 
-	/* User area size */
```

Comment:
This change is done already in 2.0.

Reply:
@nagarahalli Any reason why we shouldn't have it in master?

Reply:
It is about 2.0 merge to master, it will happen sometime in the future.

Reply:
This is not planned untill TigerMoth.

Reply:
2.0 is based on api-next, not master. This will be one of several conflicts that will need to be resolved at merge time.